ui/src/tempVars/components/TemplateMetaQueryPreview.tsx

```diff
@@ -56,7 +57,12 @@ class TemplateMetaQueryPreview extends PureComponent<Props> {
         <p className="temp-builder--validation">
           Meta Query returned <strong>{items.length}</strong> value{pluralizer}
         </p>
-        {items.length > 0 && <TemplatePreviewList items={items} />}
+        {items.length > 0 && (
```

Comment:
What do you think about making this into a getter to keep logic like this out of the render function?

Reply:
Turns out this is not needed.